packages/react-native/ReactCommon/react/renderer/mapbuffer/MapBufferBuilder.cpp

```diff
@@ -15,6 +15,7 @@ namespace facebook::react {
 constexpr uint32_t INT_SIZE = sizeof(uint32_t);
 constexpr uint32_t DOUBLE_SIZE = sizeof(double);
 constexpr uint32_t MAX_BUCKET_VALUE_SIZE = sizeof(uint64_t);
+constexpr uint64_t LONG_SIZE = sizeof(uint64_t);
```

Comment:
nits:
1. This be typed `uint32_t`
2. It should be placed next to int/double sizes